prost-build/src/lib.rs

```diff
@@ -254,6 +255,14 @@ impl Config {
         self
     }
 
+    /// Include a constant byte array containing the encoded `FileDescriptorProto` named
+    /// `FILE_DESCRIPTOR` in each generated module file. This can be used in conjunction with
+    /// the types in the `prost-types` crate to help implement server reflection.
+    pub fn file_descriptor(&mut self, include: bool) -> &mut Self {
```

Comment:
Unsolicited feedback: perhaps this should be the following?
```rust
pub fn include_file_descriptor(&mut self) -> &mut Self {
    self.file_descriptor = true;
    self
}
````

That would make it more consistent with retain_enum_prefix and compile_well_known_types

Reply:
Yes, that's probably better. I'll update.

Reply:
I've fixed this up in cebed9d - thanks for the feedback!